apps/heft/src/pluginFramework/HeftTask.ts

```diff
@@ -15,17 +15,20 @@ import type {
 } from '../utilities/CoreConfigFiles';
 import type { IHeftTaskPlugin } from '../pluginFramework/IHeftPlugin';
 import type { IScopedLogger } from '../pluginFramework/logging/ScopedLogger';
+import copyFilesOptionsSchema from '../schemas/copy-files-options.schema.json';
```

Comment:
This will collide with the change to remove `taskEvent`, though that can just outright delete this file.